packages/xo-web/src/common/selectors.js

```diff
@@ -270,7 +272,11 @@ const _getPermissionsPredicate = invoke(() => {
     }
   )
 
-  return state => {
+  return (state, props) => {
+    if (props !== undefined && props.resourceSets !== undefined) {
```

Comment:
We still want to block it if the user is not connected though. Merge this with the 'admin' `if` after the `user` check and add a comment to explain why we do that.